### ISSUE TITLE
Reuse graphs from radare2 in 'dmhg' output

### ISSUE THREAD

**Issue by XVilka** [radare/radare2#7357]
For beauty.

**Comment by radare** [comment]
Maybe add keys in the graph view to see those graphs from the interactive view? using the 'b' key to browser stuff?

**Comment by XVilka** [comment]
@radare yes, having interactive mode would be even better. And integrate with panels @Vane11ope

**Comment by radare** [comment]
the thing is to use the commands instead of the api to simplify the maintanance and reduce codebase imho

**Comment by ret2libc** [comment]
This issue has been moved from radareorg/radare2 to radareorg/ideas as we are trying to clean our backlog and this issue has probably been created a long while ago. This is an effort to help contributors understand what are the actionable items they can work on, prioritize issues better and help users find active/duplicated issues more easily. If this is not an enhancement/improvement/general idea but a bug, feel free to ask for re-transfer to main repo. Thanks for your understanding and contribution with this issue.
